src/css/global/icons.scss

```diff
@@ -199,9 +196,18 @@
 .icon-Email:before {
   content: "\e93c";
 }
+.icon-Copy:before {
+  content: "\e93d";
+}
 .icon-Twitter:before {
   content: "\ea96";
 }
-.icon-Copy:before {
-  content: "\e93d";
+.icon-Edit:before {
+  content: "\1f323";
+}
+.icon-Heart:before {
+  content: "\e93e";
 }
+.icon-Info:before {
```

Comment:
I'm not sure why these still ended up getting reordered, I used the latest icons svg from the `dev` branch. Thinking I'll leave them, if that's okay with y'all.

Reply:
It looks fine to me. The order they appear in the CSS doesn't matter and the content keys for the existing icons don't seem to have shifted. 👍